support-services/build.sbt

```diff
@@ -5,6 +5,8 @@ name := "support-services"
 description := "Scala library to provide shared services to Guardian Support projects."
 
 libraryDependencies ++= Seq(
+  "software.amazon.awssdk" % "dynamodb" % "2.13.26",
+  "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.1",
```

Comment:
new dep but it doesn't have any onward deps https://mvnrepository.com/artifact/org.scala-lang.modules/scala-java8-compat_2.12/0.9.1